### ISSUE TITLE
Generate PSPL with parallax lightcurves.

### ISSUE THREAD

**Issue by jluastro** [jluastro/PopSyCLE#51]
This is easy to do with BAGLE. We could add tutorials or we could add helpful functions inside PopSyCLE.

**Comment by jluastro** [comment]
Right now, light curves are only generated for binary/multiple objects. People write their own wrapper code generally to take PopSyCLE output and send it to BAGLE to make light cures. We could make this process easier inside of BAGLE.  The reason we don't do it by default is it can be computationally expensive. 

Maybe start with some examples.

**Comment by jluastro** [comment]
@nsabrams I don't think this is true anymore in the later functions, right? Do we have an example that can address this?

**Comment by SophieBudzik** [comment]
@jluastro @nsabrams Can you tell me which function to use? I'm interested in this feature as well

**Comment by nsabrams** [comment]
@jluastro @SophieBudzik I've added an initial tutorial here: https://github.com/MovingUniverseLab/PopSyCLE/blob/dev/docs/simulate_lightcurves_from_PopSyCLE.ipynb
